tests/v2/test_wakunode_store.nim

```diff
@@ -29,172 +29,185 @@ import
 
 from std/times import getTime, toUnixFloat
 
+
+const 
+  DefaultPubsubTopic = "/waku/2/default-waku/proto"
+  DefaultContentTopic = ContentTopic("/waku/2/default-content/proto")
+
+proc now(): Timestamp = 
+  getNanosecondTime(getTime().toUnixFloat())
+
 proc newTestMessageStore(): MessageStore =
   let database = SqliteDatabase.init("", inMemory = true)[]
   SqliteStore.init(database).tryGet()
 
+proc fakeWakuMessage(
+  payload = "TEST-PAYLOAD",
+  contentTopic = DefaultContentTopic, 
+  ts = now()
```

Comment:
Maybe add the ephemeral flag here?

Reply:
We need to extract all these test helper procedures and constants in a common module under the `tests` folder to reuse them. This is a big TODO 😄 

- [ ] Extract all test helper procedures and constants in a common module

Reply:
Agreed! should we track it in an issue?

Reply:
I have created an issue: https://github.com/status-im/nwaku/issues/1173